@xen-orchestra/web-core/lib/icons/ds-object-icon.ts

```diff
@@ -0,0 +1,284 @@
+import { defineIcon } from '@core/packages/icon'
+import { defineIconPack } from '@core/packages/icon/define-icon-pack.ts'
+import {
+  faArchive,
+  faArrowLeft,
+  faBoxesStacked,
+  faCircle,
+  faCity,
+  faClock,
+  faDatabase,
+  faDesktop,
+  faHdd,
+  faNetworkWired,
+  faPlay,
+  faSatellite,
+  faServer,
+  faSlash,
+  faUserCircle,
+  faUsers,
+  type IconDefinition,
+} from '@fortawesome/free-solid-svg-icons'
+import { dsStatusIcon } from './ds-status-icon'
+
+// type this
+function CircleStatus(status: keyof typeof dsStatusIcon): any {
+  return [
+    {
+      icon: faCircle,
+      color: 'var(--color-neutral-background-primary)',
+      translate: [7, 5.5],
+      size: 13,
+    },
+    {
+      icon: dsStatusIcon[status],
+      translate: [7, 5.5],
+      size: 10,
+    },
+  ]
+}
+
+function constructIcon(icon: IconDefinition): any {
+  return {
+    icon,
+    color: 'var(--color-neutral-txt-primary)',
+  }
+}
+
+const slash = defineIcon([
+  {
+    icon: faSlash,
+    color: 'var(--color-neutral-background-primary)',
+    translate: [-0.5, 0.5],
+    size: 20,
+  },
+  {
+    icon: faSlash,
+    color: 'var(--color-neutral-txt-secondary)',
+    translate: [0.5, -0.5],
+    size: 20,
+  },
+])
+
+const runningNeutral = defineIcon([
+  {
+    icon: faCircle,
+    color: 'var(--color-neutral-background-primary)',
+    translate: [7, 5.5],
+    size: 13,
+  },
+  {
+    icon: faCircle,
+    color: 'var(--color-neutral-txt-primary)',
+    translate: [7, 5.5],
+    size: 10,
+  },
+  {
+    icon: faPlay,
+    color: 'var(--color-neutral-background-primary)',
+    translate: [7.5, 5.5],
+    size: 5,
+  },
+])
+
+const retrun = defineIcon([
```

Comment:
What do you mean ? 
```suggestion
const rerun = defineIcon([
```
or
```suggestion
const return = defineIcon([
```
??